docs/tutorials/ServerModule/ServerModule.md

```diff
@@ -101,25 +115,21 @@ Even the greatest ServerModule must be stopped from time to time. We must overri
 /// </summary>
 protected override void OnStop()
 {
-    // Stop wcf hosts 
-    foreach (var host in _hosts)
-        host.Dispose();
-
-    _hosts.Clear();
-
     // Deactivate facades
     DeactivateFacade(_playGroundExecution);
 
+    // Stop Plugin
+    Container.Resolve<IExamplePlugin>().Stop();
 }
 ````
 
-## The ModuleConfig.cs - File <a id="moduleConfigFile" />
+## The ModuleConfig.cs - File 
 
```

Comment:
For the the following paragraph:
- During the _build_ process an xml..
- _ModuleConfig.cs_. Here you ..
- Furthermore, you ..